tests/testthat/test_20-coupled.R

```diff
@@ -261,17 +263,20 @@ for (csvfile in csvfiles) {
       getLine = function() stop("getLine should not called."),
       .package = "gms"
     )
-    unlink("../../*TESTTHAT.RData")
+    unlink("../../*TESTTHAT.RData", force = TRUE)
     expect_no_warning(config <- readCheckScenarioConfig(csvfile, remindPath <- file.path("..", "..")))
   })
 }
 
 test_that("delete files to leave clean state", {
   # leave clean state
   skipIfPreviousFailed()
-  if (! all(file.exists(deleteallfiles))) {
+  missingfiles <- setdiff(deleteallfiles[! file.exists(deleteallfiles)],
+                          paste0("../../output/C_TESTTHAT-SSP2-NDC-rem-", seq(2)))
```

Comment:
Why?

Reply:
Because the NDC folders are deleted [here](https://github.com/remindmodel/remind/pull/1998/files#diff-fac9a219359b375c544f474aa3daebc318fed1ac7d6b9a5afbcfb4531c8b2027R208) to see if everything fails correctly if mag iteration is bigger than rem iteration. And so you cannot expect them to exist at this point of the code.